compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/tree/SyntaxKind.java

```diff
@@ -50,8 +50,11 @@
     WHILE_KEYWORD(207, "while"),
     CHECK_KEYWORD(208, "check"),
     CHECKPANIC_KEYWORD(209, "checkpanic"),
+    CONTINUE_KEYWORD(212,"continue"),
+    BREAK_KEYWORD(213,"break"),
```

Comment:
seems this is not in the correct order. Also next number should be 211

Reply:
I'll fix that

Reply:
done